### PR TITLE
Add ol3-geocoder

### PR DESCRIPTION
**Pull request by jonataswalker**

### CODE REVIEW

**Comment by jimaek** [comment]
If this is an openlayers plugin then please rename the directory to `openlayers.geocoder`

File: files/ol3-geocoder/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "ol3-geocoder",
+  "repo": "jonataswalker/ol3-geocoder",
+  "files": {
+    "include": ["ol3-geocoder.js", "ol3-geocoder-debug.js", "ol3-geocoder.css"]
```
Comment (jimaek) [inline files/ol3-geocoder/update.json]: I dont think `ol3-geocoder-debug.js` is required in production. Can you remove it?

**Comment by jonataswalker** [comment]
Hi @jimaek, changes made. Thanks for your attention.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**